src/common/components/Breadcrumbs/breadcrumbs.less

```diff
@@ -0,0 +1,11 @@
+@import "../../styles/variables.less";
+
+.jdn {
+  &__breadcrumb {
+    overflow-wrap: anywhere;
+
+    span:first-of-type {
+      color: @breadcrumb-first-child;
```

Comment:
instead of writing a style, I'd recommend to use ant's components for text formatting: `<Text type="secondary">{wrappedText}</Text>`. It will allow us to use default ant's colors, that Semyon also using for mockups